<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telephonePad</class>
 <widget class="QDialog" name="telephonePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telephone Pad</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QToolButton" name="button1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    width:51;
    hieght:51;
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="button2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="button3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="button4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="button5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="button6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="button7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="button8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="button9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="button10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>#</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="button0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="button11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QToolButton" name="buttonClear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>AC</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icon/backspace-arrow.png</normaloff>icon/backspace-arrow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>47</width>
           <height>47</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QToolButton" name="buttonCall">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QToolButton:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
         </property>
         <property name="text">
          <string>Call</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lcdNumber">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border-style: outset;
    border-width: 1px;
    border-radius: 10px;
    border-color: beige;
    font: bold 47px;
    
     
    padding: 6px;
    background-color: #000000;
    color:#1976D2;
   
}
QLineEdit:pressed {
    background-color: gray;
    border-style: inset;
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
